### PR TITLE
Use `StringNumber` for integers beyond `i32` - part1

### LINKED ISSUE #260: Fix Integer overflow at graphql API response - part1
#37 와 관련하여 conn, dcerpc, dns, ftp 부분 한정하여 개선하려고 합니다.

대상
graphql
- event/conn.rs
- event/dcerpc.rs
- event/dns.rs
- event/ftp.rs

### PR DESCRIPTION
**Pull request by henry0715-dev**
Close #260

### CODE REVIEW

**Comment by sehkone** [comment]
If this is part 1, what would we call #257?

**Comment by henry0715-dev** [comment]
> If this is part 1, what would we call #257?

part를 나누기 이전에 작업을 했던 부분입니다.
part0로 설정하는것은 어떨까요?

**Comment by sehkone** [comment]
> > If this is part 1, what would we call #257?
> 
> part를 나누기 이전에 작업을 했던 부분입니다. part0로 설정하는것은 어떨까요?

Sounds good. Any would be fine if it fits the pattern.

File: src/graphql/event/conn.rs [hunk 1]
```diff
@@ -374,28 +374,33 @@ impl BlockListConn {
...
+        StringNumber(self.inner.duration)
     }
 
     async fn service(&self) -> String {
         self.inner.service.clone()
     }
 
-    async fn orig_bytes(&self) -> u64 {
-        self.inner.orig_bytes
+    /// The `orig bytes` in bytes within the range representable by a `u64`.
```
Comment (sehkone) [inline src/graphql/event/conn.rs:386]: `orig bytes` enclosed in backticks is not actual code. We should use a more descriptive phrase without backticks.

Comment (sehkone) [reply src/graphql/event/conn.rs:386]: The others should be corrected the same way.

**Comment by sehkone** [comment]
@henry0715-dev
Could you rewrite the body of the commit message? It repeats the same as the title and its format doesn't seem to follow our guideline.

cc: @sophie-cluml

**Comment by henry0715-dev** [comment]
> @henry0715-dev
> 
> Could you resolve the conflicts by rebasing this?

It's done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -50,7 +50,9 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   - Removed `get_node_settings` function as it is no longer used.
   - Renamed `get_customer_id_of_review_host` to `get_customer_id_of_node`.
 - Updated review-database to 0.29.1.
-- In `BlockListSmb`, `Network`, `SamplingPolicy`, `Tidb`, and `TidbRule`,
+- In `BlockListSmb`, `Network`, `SamplingPolicy`, `Tidb`, `TidbRule`,
+  `BlockListConn`, `BlockListDceRpc`, `DnsCovertChannel`, `LockyRansomware`,
+  `CryptocurrencyMiningPool`, `BlockListDns`, `FtpPlainText` AND `BlockListFtp`
```
Comment (sehkone) [inline CHANGELOG.md:55]: `AND` should be lowercase.

Comment (henry0715-dev) [reply CHANGELOG.md]: I modified it.
I accidentally went into the upper case, but I didn't recognize it.
Thank you for your opinion.

**Review by sehkone — APPROVED** [review]

**Comment by sehkone** [comment]
@henry0715-dev,

I am sorry for asking you to do one more rebase.

**Comment by henry0715-dev** [comment]
> @henry0715-dev,
> 
> I am sorry for asking you to do one more rebase.

I have completed the reflection.
Thank you.

**Review by sehkone — APPROVED** [review]

**Pull request merged**